src/wp-admin/user-edit.php

```diff
@@ -796,8 +797,8 @@
 			$application_passwords_list_table->display();
 			?>
 		</div>
-		<?php else : ?>
-			<p><?php _e( 'The application password feature requires HTTPS, which is not enabled on this site.' ); ?></p>
+		<?php elseif ( ! wp_is_application_passwords_supported() ) : ?>
+			<p><?php _e( 'The application password feature requires HTTPS - which is not enabled on this site - or the environment type set as local.' ); ?></p>
```

Comment:
Restoring the string change as it could confuse users who won't understand what a local env type is.

For local development, tools like `wp-env` or Local automatically set the `'local'` env type.

```suggestion
			<p><?php _e( 'The application password feature requires HTTPS , which is not enabled on this site.' ); ?></p>
```